lib/ansible/utils/encrypt.py

```diff
@@ -270,12 +270,13 @@ def _hash(self, secret, salt, salt_size, rounds, ident):
 
 
 def passlib_or_crypt(secret, algorithm, salt=None, salt_size=None, rounds=None, ident=None):
+    display.deprecated("passlib_or_crypt API is deprecated in favor of do_encrypt", version='2.20')
+    return do_encrypt(secret=secret, algorithm=algorithm, salt=salt, salt_size=salt_size, rounds=rounds, ident=ident)
+
+
+def do_encrypt(secret, algorithm, salt=None, salt_size=None, rounds=None, ident=None):
```

Comment:
The order and names of the existing function arguments should remain the same as before. If we're going to change the names or order, a new function should be created with those arguments and the existing function deprecated.

Reply:
Ok. make sense.